.bazelrc

```diff
@@ -61,9 +61,10 @@ build:asan --features=asan
 
 # Configuration for enabling LibFuzzer (along with ASan).
 build:fuzzer --features=fuzzer
-
-# Proto fuzzer specific configuration.
-build:proto-fuzzer --features=proto-fuzzer
+build:fuzzer --features=-libc++
+build:fuzzer --@rules_fuzzing//fuzzing:cc_engine=@rules_fuzzing//fuzzing/engines:libfuzzer
```

Comment:
Can we not point this at our build rule for libfuzzer to preserve that?

I'm somewhat sad to lose the really broad consistency of using Clang + libc++ everywhere we can, and having weird differences that crop up because of the oddities of the C++ standard library we happen to be linking...

I guess if this is the only way to make oss-fuzz work, but it seems like ... a really big thing to give up.

Reply:
The outputs are quite a bit different. If you want to maintain separate rules, I would abandon this change, but probably also leave oss-fuzz to someone else to figure out how it's could work. The inability use the provided bazel integration point will increase complexity, and I'm trying not to balance how much time I'm spending here. We can always leave oss-fuzz integration for later.

Reply:
I was hoping they were more similar. :sigh: All of the Bazel stuff in `rules_fuzzing` is a bit scary in how complex it has gotten compared to our setup.

That said, I'm not really objecting. I'm mostly nervous. My hope with our build set up was to really minimize the moving pieces we have to support, and this adds a big new one.

But as you point out, this seems likely the only easy path to having OSS-Fuzz working, and this config is primarily useful in that context. So, if this works to get OSS-Fuzz working, it's almost certainly the right call.

Maybe worth leaving a comment that this config isn't supported everywhere and may be brittle, but is designed to roughly match the continuous fuzzing we get from OSS-fuzz?